### PR TITLE
Add quickstart resources

### PR DESCRIPTION
**Pull request by dominicbetts**
## Purpose

Add config files used by the latest version of the quickstarts

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/quickstarts/dataflow.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  name: mq-to-kafka
+  namespace: azure-iot-operations
+spec:
+  profileRef: example-dataflow
+  operations:
+    - operationType: source
+      sourceSettings:
+        endpointRef: mq-source
+        dataSources:
+        - $share/group/azure-iot-operations/data/thermostat
```
Comment (jlian) [inline samples/quickstarts/dataflow.yaml:51]: I think we made an update to remove the need for `$share/group` part when `instanceCount` is 1.

File: samples/quickstarts/event-hubs-config.bicep [hunk 2]
```diff
@@ -0,0 +1,41 @@
+@description('Location for cloud resources')
+param aioExtensionName string = 'azure-iot-operations'
+param clusterName string = 'clusterName'
+param eventHubNamespaceName string = 'default'
```
Comment (jlian) [inline samples/quickstarts/event-hubs-config.bicep:4]: I guess the correct params will be passed in via command line?

File: samples/quickstarts/kafka-connector.yaml
Comment (jlian) [inline samples/quickstarts/kafka-connector.yaml:1]: This file isn't needed, right?

File: samples/quickstarts/dataflow.yaml (hunk 1 above)
Comment (dominicbetts) [reply samples/quickstarts/dataflow.yaml:51]: It's working as is right now - I'll check again tomorrow.

File: samples/quickstarts/event-hubs-config.bicep (hunk 2 above)
Comment (dominicbetts) [reply samples/quickstarts/event-hubs-config.bicep:4]: Yes

**Pull request merged**